src/sources/mod.rs

```diff
@@ -106,144 +106,180 @@ enum BuildError {
 #[serde(tag = "type", rename_all = "snake_case")]
 #[enum_dispatch(SourceConfig)]
 pub enum Sources {
-    /// AMQP.
+    /// Collect events from AMQP 0.9.1 compatible brokers like RabbitMQ.
     #[cfg(feature = "sources-amqp")]
+    #[configurable(metadata(docs::label = "AMQP"))]
     Amqp(amqp::AmqpSourceConfig),
 
-    /// Apache HTTP Server (HTTPD) Metrics.
+    /// Collect metrics from Apache's HTTPD server.
     #[cfg(feature = "sources-apache_metrics")]
+    #[configurable(metadata(docs::label = "Apache Metrics"))]
     ApacheMetrics(apache_metrics::ApacheMetricsConfig),
 
-    /// AWS ECS Metrics.
+    /// Collect Docker container stats for tasks running in AWS ECS and AWS Fargate.
     #[cfg(feature = "sources-aws_ecs_metrics")]
+    #[configurable(metadata(docs::label = "AWS ECS Metrics"))]
     AwsEcsMetrics(aws_ecs_metrics::AwsEcsMetricsSourceConfig),
 
-    /// AWS Kinesis Firehose.
+    /// Collect logs from AWS Kinesis Firehose.
     #[cfg(feature = "sources-aws_kinesis_firehose")]
+    #[configurable(metadata(docs::label = "AWS Kinesis Firehose"))]
     AwsKinesisFirehose(aws_kinesis_firehose::AwsKinesisFirehoseConfig),
 
-    /// AWS S3.
+    /// Collect logs from AWS S3.
     #[cfg(feature = "sources-aws_s3")]
+    #[configurable(metadata(docs::label = "AWS S3"))]
     AwsS3(aws_s3::AwsS3Config),
 
-    /// AWS SQS.
+    /// Collect logs from AWS SQS.
     #[cfg(feature = "sources-aws_sqs")]
+    #[configurable(metadata(docs::label = "AWS SQS"))]
     AwsSqs(aws_sqs::AwsSqsConfig),
 
-    /// Datadog Agent.
+    /// Receive logs, metrics, and traces collected by a Datadog Agent.
     #[cfg(feature = "sources-datadog_agent")]
+    #[configurable(metadata(docs::label = "Datadog Agent"))]
     DatadogAgent(datadog_agent::DatadogAgentConfig),
 
-    /// Demo logs.
+    /// Generate fake log events, which can be useful for testing and demos.
     #[cfg(feature = "sources-demo_logs")]
+    #[configurable(metadata(docs::label = "Demo Logs"))]
     DemoLogs(demo_logs::DemoLogsConfig),
 
-    /// DNSTAP.
+    /// Collect DNS logs from a dnstap-compatible server.
     #[cfg(all(unix, feature = "sources-dnstap"))]
+    #[configurable(metadata(docs::label = "dnstap"))]
     Dnstap(dnstap::DnstapConfig),
 
-    /// Docker Logs.
+    /// Collect logs from Docker.
```

Comment:
```suggestion
    /// Collect container logs from a Docker Daemon.
```

**suggestion**: Clarifies the logs being collected are from the containers run by a docker daemon, rather the daemon's logs.